client/my-sites/ads/form-settings.jsx

```diff
@@ -267,6 +279,75 @@ class AdsFormSettings extends Component {
 		);
 	}
 
+	displayOptions() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<FormFieldset>
+					<FormLegend>{ translate( 'Display ads below posts on' ) }</FormLegend>
+					<CompactFormToggle
+						checked={ !! this.state.display_options.display_front_page }
+						disabled={ this.state.isLoading }
+						onChange={ this.handleDisplayToggle( 'display_front_page' ) }
+					>
+						{ translate( 'Front page' ) }
+					</CompactFormToggle>
+					<br />
```

Comment:
It's a little odd that we need `<br />`'s here

Reply:
I would suggest modifying styling here with CSS instead.

Reply:
Thanks for the note @gwwar, I was pulling my hair out trying to figure out why these CompactFormToggles were all lining up on the same line when other examples using them don't do that. I'll go back and see if I can fix it with the CSS.

<img width="765" alt="screen shot 2017-11-16 at 4 04 02 pm" src="https://user-images.githubusercontent.com/273708/32922608-dd1c6712-cae7-11e7-9553-d50263090388.png">